### PR TITLE
switch from dependency to package

### PR DESCRIPTION
**Pull request by gadfort**

### CODE REVIEW

File: .github/workflows/python_ci.yml [hunk 1]
```diff
@@ -29,7 +29,9 @@ jobs:
         if: matrix.os == 'windows-latest'
         run: |
           choco install -y graphviz
-
+          New-ItemProperty -Path "HKLM:\SYSTEM\CurrentControlSet\Control\FileSystem" -Name "LongPathsEnabled" -Value 1 -PropertyType DWORD -Force
```
Comment (troibe) [inline .github/workflows/python_ci.yml:32]: I did see random failures on Windows if it was due to long paths then thanks so much for fixing that.

File: siliconcompiler/package.py [hunk 2]
```diff
@@ -12,74 +12,48 @@
...
-            or
-            'path': e.g. 'https://github.com/xyz/xyz/archive/',
-            'ref': e.g. '3b94aa80506d25d5388131e9f2ecfcf4025ca866'
-            or
-            'path': e.g. 'https://zeroasic.com/xyz.tar.gz',
-            'ref': e.g. 'your-dependency-version-1'
-            or
-            'path': e.g. 'python://siliconcompiler',
+        package (str): Name of the data source
+        quiet (boolean): Suppress error messages
```
Comment (troibe) [inline siliconcompiler/package.py:19]: Suppress package found messages

File: tests/package/test_package.py [hunk 3]
```diff
@@ -1,17 +1,26 @@
...
+from siliconcompiler import package
 from pathlib import Path
 import pytest
 import logging
+import os
 
 
-def dependency_cache_path(path, ref, chip=None, quiet=True):
+@pytest.fixture(autouse=True)
+def mock_home(monkeypatch):
```
Comment (troibe) [inline tests/package/test_package.py:10]: What were your thoughts for adding this?

Comment (gadfort) [reply tests/package/test_package.py:10]: I prefer the tests not modify my local setup and run in the pytest directory

Comment (troibe) [reply tests/package/test_package.py:10]: Sounds reasonable. Should we do that globally then?
Any test that uses a target will download or use the cache as well.

Comment (gadfort) [reply tests/package/test_package.py:10]: Good point, can you handle that? just push the changes to this PR

**Pull request merged**